src/components/LoansOnOfferComponent.js

```diff
@@ -0,0 +1,112 @@
+import  React, { Component } from "react";
+import AuthService from "../services/authService";
+
+import house from '../house.jpg';
+import car from '../car.jpg'
+import personal from '../personal.jpg'
+
+let nowDate = new Date(); 
+
+
+
+
+export default class LoansOnOfferComponent extends Component {
+    constructor(props) {
+        super(props);
+        this.handleSubmitHomeLoan = this.handleSubmitHomeLoan.bind(this);
+        this.handleSubmitAutoLoan = this.handleSubmitAutoLoan.bind(this);
+        this.handleSubmitPersonalLoan = this.handleSubmitPersonalLoan.bind(this);
+
+    }
+    
+
+     handleSubmitHomeLoan() {
+        AuthService.loanSignup(530000.12, 'Home', 2.50, 120000.23, nowDate.getDate())
+        .then(
+        () => {
+            this.props.history.push("/signupLoan");
+            window.location.reload();                   
+
+        })
+      }
+
+      handleSubmitAutoLoan() {
+        AuthService.loanSignup(12000.60, 'Auto', 3.70, 5000.12, nowDate.getDate())
+        .then(
+        () => {
+            this.props.history.push("/signupLoan");
+            window.location.reload();                        
+
+        })
+      }
+
+      handleSubmitPersonalLoan() {
+        AuthService.loanSignup(5000.10, 'Personal', 4.10, 1000.34, nowDate.getDate())
+        .then(
+        () => {
+            this.props.history.push("/signupLoan");
+            window.location.reload();                        
+
+        })
+      }
```

Comment:
This ends up registering a new loan type into the database every time a user signs up for one...

Reply:
Again, this is the same issue you brought up in the backend MS, which I explained there...